packages/render-fidelity-tools/src/components/renderers/babylon-viewer.ts

```diff
@@ -114,17 +114,18 @@ export class BabylonViewer extends LitElement {
 
     this[$scene].stopAllAnimations();
 
-    // load hdr directly (the size of cubmap is set to be 256 for all renderers)
+    // load and prefilter HDR texture (the size of cubmap is set to be 256 for
+    // all renderers)
     const environment = new HDRCubeTexture(
-        scenario.lighting, this[$scene], 256, false, false, false);
+        scenario.lighting, this[$scene], 256, false, false, false, true);
```

Comment:
In latest version of Babylon, the HDRCubeTexture's constructor has some more options: 
noMipmap?: boolean, generateHarmonics?: boolean, gammaSpace?: boolean, prefilterOnLoad?: boolean.
I don't know if i need to set the other three variables to true or not, so i just leave them to be false.

Reply:
We should experiment a little with these in a following PR to see what gives the closest results.